docs/guides/qiskit-addons-cutting-gates.ipynb

```diff
@@ -0,0 +1,433 @@
+    "with Batch(backend=backend) as batch:\n",
+    "    sampler = SamplerV2(mode=batch)\n",
+    "    jobs = {\n",
+    "        label: sampler.run(subsystem_subexpts, shots=2**12)\n",
+    "        for label, subsystem_subexpts in isa_subexperiments.items()\n",
+    "    }\n",
+    "\n",
+    "# Retrieve results\n",
+    "results = {label: job.result() for label, job in jobs.items()}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "64e762ad-0660-4c78-8ea7-fb9d226357d7",
+   "metadata": {},
+   "source": [
+    "Lastly the expectation value of the full circuit can be reconstructed using the [`reconstruct_expectation_values`](../api/qiskit-addon-cutting/qiskit-addon-cutting#reconstruct_expectation_values) method.\n",
+    "\n",
+    "The code block below reconstructs the results and compares them with the exact expectation value."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "id": "b3cf1b65-df16-4bd4-a083-f65cec6c49dc",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Reconstructed expectation value: 0.59146589\n",
+      "Exact expectation value: 0.56254612\n",
+      "Error in estimation: 0.02891977\n",
+      "Relative error in estimation: 0.0514087\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Get expectation values for each observable term\n",
+    "reconstructed_expval_terms = reconstruct_expectation_values(\n",
+    "    results,\n",
+    "    coefficients,\n",
+    "    subobservables,\n",
+    ")\n",
+    "\n",
+    "# Reconstruct final expectation value\n",
+    "reconstructed_expval = np.dot(reconstructed_expval_terms, observable.coeffs)\n",
+    "\n",
+    "\n",
+    "estimator = EstimatorV2()\n",
+    "exact_expval = estimator.run([(qc, observable)]).result()[0].data.evs\n",
+    "print(\n",
+    "    f\"Reconstructed expectation value: {np.real(np.round(reconstructed_expval, 8))}\"\n",
+    ")\n",
+    "print(f\"Exact expectation value: {np.round(exact_expval, 8)}\")\n",
+    "print(\n",
+    "    f\"Error in estimation: {np.real(np.round(reconstructed_expval-exact_expval, 8))}\"\n",
+    ")\n",
+    "print(\n",
+    "    f\"Relative error in estimation: {np.real(np.round((reconstructed_expval-exact_expval) / exact_expval, 8))}\"\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1a04014c-a9a4-42e6-b268-d5c8fe7b2b94",
+   "metadata": {},
+   "source": [
+    "## Gate cutting to reduce circuit depth\n",
+    "\n",
+    "Next we'll demonstrate a workflow which reduces a circuit's depth by cutting distant gates, avoiding a large series of swap gates that would otherwise be introduced.\n",
+    "\n",
+    "We'll start first with the [`EfficientSU2`](../api/qiskit/qiskit.circuit.library.EfficientSU2) ansatz, but with \"circular\" entanglement in order to introduce distant gates. We'll also define the same observable to measure as the previous example."
```

Comment:
```suggestion
    "The following workflow reduces a circuit's depth by cutting distant gates, avoiding a large series of swap gates that would otherwise be introduced.\n",
    "\n",
    "Start with the [`EfficientSU2`](/api/qiskit/qiskit.circuit.library.EfficientSU2) ansatz, but with \"circular\" entanglement in order to introduce distant gates. Here you will define the same observable to measure as the previous example."
```